content/docs/guides/first_steps.md

```diff
@@ -0,0 +1,173 @@
+
+The `rule_files` block specifies the location of any rules we want the Prometheus server to load. For now we've got no rules.
+
+The last block, `scrape_configs`, controls what resources Prometheus monitors. Since Prometheus also exposes data about itself as an HTTP endpoint it can scrape and monitor its own health. In the default configuration there is a single job, called `prometheus`, which scrapes the time series data exposed by the Prometheus server. The job contains a single, statically configured, target, the `localhost` on port `9090`. Prometheus expects metrics to be available on targets on a path of `/metrics`. So this default job is scraping via the URL: http://localhost:9090/metrics.
+
+The time series data returned will detail the state and performance of the Prometheus server.
+
+For a complete specification of configuration options, see the
+[configuration documentation](/docs/operating/configuration).
+
+## Starting Prometheus
+
+To start Prometheus with our newly created configuration file, change to the directory containing the Prometheus binary and run:
+
+```language-bash
+./prometheus --config.file=prometheus.yml
+```
+
+Prometheus should start up. You should also be able to browse to a status page about itself at http://localhost:9090. Give it about 30 seconds to collect data about itself from its own HTTP metrics endpoint.
+
+You can also verify that Prometheus is serving metrics about itself by
+navigating to its own metrics endpoint: http://localhost:9090/metrics.
+
+## Using the expression browser
+
+Let us try looking at some data that Prometheus has collected about itself. To
+use Prometheus's built-in expression browser, navigate to
+http://localhost:9090/graph and choose the "Console" view within the "Graph"
+tab.
+
+As you can gather from http://localhost:9090/metrics, one metric that
+Prometheus exports about itself is called
+`http_requests_total` (the total number of HTTP requests the Prometheus server has made). Go ahead and enter this into the expression console:
+
+```
+http_requests_total
+```
+
+This should return a number of different time series (along with the latest value recorded for each), all with the metric name `http_requests_total`, but with different labels. These labels designate different types of requests.
+
+If we were only interested in requests that resulted in HTTP code `200`, we could use this query to retrieve that information:
+
+```
+http_requests_total{code="200"}
+```
+
+To count the number of returned time series, you could write:
+
+```
+count(http_requests_total)
+```
+
+For more about the expression language, see the
+[expression language documentation](/docs/querying/basics/).
+
+## Using the graphing interface
+
+To graph expressions, navigate to `http://localhost:9090/graph` and use the "Graph" tab.
```

Comment:
Sometimes you use backticks on urls, other times you don't. It would be good to be consistent.